### PR TITLE
Moby Dev Report 2017 06 05

### PR DESCRIPTION
**Pull request by vieux**
ping @tonistiigi @tiborvass @thaJeztah

### CODE REVIEW

File: reports/2017-06-05.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Development Report for June 5, 2017
+
+## Daily Meeting
+
+A daily meeting is hosted on [slack](https://dockercommunity.slack.com) every business day at 9am PST on the channel `#moby-project`.
+Lots of discussion happened during this meeting to kickstart the project, but now that we have the forums, we see less activity there.
+Ee are discussing the future of this meeting [here](https://forums.mobyproject.org/t/of-standups-future), we will possibily move the meeting
```
Comment (thaJeztah) [inline reports/2017-06-05.md]: s/Ee/We/

Comment (AkihiroSuda) [inline reports/2017-06-05.md]: s/Ee/We

**Review by AkihiroSuda — COMMENTED** [review]
6-5 seems a dupe of 5-29

File: reports/2017-06-05.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+### Builder
+
+The builder dev report can be found [here](builder/2017-06-05.md)
+
+
+### LinuxKit
+
+The LinuxKit dev report can be found [here](https://github.com/linuxkit/linuxkit/blob/master/reports/2017-06-03.md_
```
Comment (AkihiroSuda) [inline reports/2017-06-05.md]: Extra underscore

File: reports/builder/2017-06-05.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Related to this, @simonferquel opened a [WIP PR](https://github.com/moby/moby/pull/33492) that introduces typed Dockerfile parsing. This enables making [decisions about dependencies](https://github.com/moby/moby/issues/32550#issuecomment-297867334) between build stages and reusing Dockerfile parsing as a buildkit frontend.
+
+### Buildkit
+
+Some initial proof of concept code for [buildkit](https://github.com/moby/moby/issues/32925) has been pushed to https://github.com/tonistiigi/buildkit_poc . It's in a very early exploratory stage. Current codebase includes libraries for getting concurrency safe references to containerd snapshots using a centralized cache management instance. There is a sample source implementation for pulling images to these snapshots and executing jobs with runc on top of them. There is also some utility code for concurrent execution and progress stream handling. More info should follow in next weeks, including hopefully opening up an official repo. If you have questions or want to help, stop by the issues section of that repo or the proposal in moby/moby.
+
+### `v17.06` release
```
Comment (thaJeztah) [inline reports/builder/2017-06-05.md]: This is not related to moby? Or should be "discovered downstream in Docker 17.06"

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Left some comments, but LGTM otherwise

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**